lib/vicadmin/validate.go

```diff
@@ -62,8 +62,9 @@ type Validator struct {
 }
 
 const (
-	GoodStatus = template.HTML(`<i class="icon-ok"></i>`)
-	BadStatus  = template.HTML(`<i class="icon-attention"></i>`)
+	GoodStatus     = template.HTML(`<i class="icon-ok"></i>`)
+	BadStatus      = template.HTML(`<i class="icon-attention"></i>`)
+	DefaultVCHName = `VCH`
```

Comment:
Logging the failure requires looking at the logs - changing the name of the VCH to something potentially misleading when someone looks at the admin console is a different matter.
Looking weird is fine - we could also set it to something obviously not a name, such as "<unable to retrieve vch name from vsphere>"